### PR TITLE
Small changes in Kubemark requested by @wojtek-t

### PR DESCRIPTION
**Pull request by gmarek**

### CODE REVIEW

File: test/kubemark/stop-kubemark.sh [hunk 1]
```diff
@@ -38,4 +39,6 @@ gcloud compute disks delete \
       --zone "${ZONE}" \
       "${MASTER_NAME}"-pd || true
 
+echo -e "${color_red}Warning: firewall-rule has not been removed. Please remove it by hand if you're not planning to run more Kubemark experiments.${color_norm}"
```
Comment (wojtek-t) [inline test/kubemark/stop-kubemark.sh]: Why not just remove it?

Comment (gmarek) [inline test/kubemark/stop-kubemark.sh]: It'll make cycle slower, but I can remove it.

**Comment by gmarek** [comment]
It'll require manual cleanup of existing resources.

**Comment by wojtek-t** [comment]
LGTM - thanks for those fixes!

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**